lib/v2/foreverblog-cn/feeds.js

```diff
@@ -0,0 +1,35 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const currentUrl = `https://www.foreverblog.cn/feeds.html`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+    const $articles = $('article[class="post post-type-normal"]');
+    const items = $articles
+        .map((_, el) => {
+            const $titleDiv = $(el).find('h1[class="post-title"]');
+            const title = $titleDiv.text().trim();
+            const link = $titleDiv.find('a').eq(0).attr('href');
+            const author = $(el).find('div[class="post-author"]').text().trim();
+            const postDate = $(el).find('time').text().trim();
+            const description = `${postDate} ${author}: ${title}`;
```

Comment:
Putting postDate in `pubDate` instead will be a better choice since user can sort feed items in their readers

Reply:
This date is really a date, E.g. "12-05", no year, no time. So I just put it in title. Let the reader App to use grab time for the pub time.
And another reason is that I'm not sure how to convert '12-31' info to '2022-12-31' or '2023-12-31'. If it's really needed, I can use the `pubDate`.

Reply:
```js
const { parseDate } = require('@/utils/parse-date');
```
docs: https://docs.rsshub.app/joinus/pub-date.html

[dayjs](https://github.com/iamkun/dayjs) will handle the rest.
The time will be 00:00 GMT. The year will be this year if it sees '12-05' and will be last year if it's '12-31' on 6th December 2022